hack/builder/titus-executor-builder.sh

```diff
@@ -117,7 +117,7 @@ if [[ $num_debs -ne 1 ]]; then
     exit 1
 fi
 
-filename=${outdir}/*.deb
+filename=$(ls -t ${outdir}/*.deb | grep -v latest | head -n 1)
```

Comment:
`-1t`

Reply:
Wait, what's wrong with selecting on latest?

Reply:
`-1` is the default when `ls` is not run in a shell.  Latest is the symlink: we want to grab the source file here.